javacorelib/src/main/scala/java/lang/System.scala

```diff
@@ -148,7 +145,7 @@ object System {
           ""
       map.put(name, value)
       ptr = ptr + 1
-    }
+    }*/
```

Comment:
This needs to be refactored to move platform-dependant parts into `runtime.Platform`.

Reply:
Yes, I am fighting with `javafilelib` at the moment, after that I will fix all this small issues about `System.scala`

Reply:
System.scala fully works now